examples/splinterdb_custom_ipv4_addr_sortcmp_example.c

```diff
@@ -160,40 +149,35 @@ main()
    splinterdb_config splinterdb_cfg;
    configure_splinter_instance(&splinterdb_cfg,
                                &splinter_data_cfg,
-                               APP_DB_NAME,
-                               (APP_DEVICE_SIZE_MB * K_MiB),
-                               (APP_CACHE_SIZE_MB * K_MiB));
+                               DB_FILE_NAME,
+                               (DB_FILE_SIZE_MB * 1024 * 1024),
+                               (CACHE_SIZE_MB * 1024 * 1024));
 
    splinterdb *spl_handle = NULL; // To a running SplinterDB instance
 
    int rc = splinterdb_create(&splinterdb_cfg, &spl_handle);
    if (rc) {
-      ex_err("SplinterDB creation failed. (rc=%d)\n", rc);
+      printf("SplinterDB creation failed. (rc=%d)\n", rc);
       return rc;
    }
 
-   ex_msg("Insert and iterate through %d {ipaddr} -> {www-url, metrics} "
+   printf("Insert and iterate through %d {ipaddr} -> {www-url, metrics} "
           "mapping table:\n",
           num_inet_addrs);
    do_inserts(spl_handle, inet_addr_info, num_inet_addrs);
 
-   do_iterate_all(spl_handle, num_inet_addrs);
+   // NULL start_key => Iterate through all key-value pairs
+   do_iterate_from(spl_handle, NULL);
```

Comment:
It was better to simplify the "iterate_all()" as a special case of `iterate_from()` by supplying a NULL `start_key` value.